components/Cards/index.js

```diff
@@ -18,22 +18,28 @@
 //
 // Create a card for each of the articles and add the card to the DOM.
 
+// Select main node
+const cardsContainer = document.querySelector('.cards-container');
+
 axios.get(`https://lambda-times-backend.herokuapp.com/articles`)
   .then(data => {
     // test response
     console.log('response', data);
 
     // hold received data
-    const articleData = data.data;
-
-    // create tab and append data
-    const cards = document.querySelector('.cards');
-    const card = createCard(articleData);
-    cards.appendChild(card);
-
-  })
-
+  const articleData = data.data.articles; // data is stored in an object
```

Comment:
always remember to look at the console logs when you are grabbing data from the backend